src/pages/BookmarksPage.tsx

```diff
@@ -35,9 +31,18 @@ function BookmarkPage({location}: RouteComponentProps) {
       </StyledHeaderWrapper>
       <Content>
         <ScheduleList>
-          {bookmarks.map(({run}) => (
-            <ScheduleCard key={run.scheduled + (run.players.join('-') || '')} run={run} />
-          ))}
+          {bookmarks.size === 0 ? (
+            <p>You don't have any bookmarks yet. Go to the schedule and favorite some!</p>
```

Comment:
`You don't have any bookmarks yet. Go to the schedule and mark some!`
Maybe this wording is better?